views/services/utils.es

```diff
@@ -67,21 +67,13 @@ export const gameReload = () => {
   } else {
     doc = document;
   }
-  var flash = doc.getElementById('flashWrap');
```

Comment:
These codes are still required for game reload function working, but it's okay to rename these variable names.

Reply:
Thx for pointing that out. I broke the game reload feature.
I fixed it and checked the operation.